.gitignore

```diff
@@ -51,6 +51,10 @@ coverage.xml
 .hypothesis/
 .pytest_cache/
 tests/books/toc/_build/
+tests/books/clean_cache/_build/
+tests/books/config/_build/
```

Comment:
I wonder if we could just do `*/_build`?

Reply:
good idea -- I'll update

Reply:
see https://github.com/executablebooks/jupyter-book/pull/626/commits/3f653e28a03e9ec281a622a073b7900efa55dc8b